src/core/pratter.ml

```diff
@@ -0,0 +1,131 @@
+(* FIXME: use opam version instead *)
+(** This modules defines a functor whose image is a parser for terms with
+    applications, binary and unary operators. These terms are specified in the
+    argument of the functor.
+
+    The algorithm implemented is an extension of the Pratt parser. The Sunting
+    Yard algorithm could also be used.
+    @see <https://dev.to/jrop/pratt-parsing>
+    @see <https://effbot.org/zone/simple-top-down-parsing.htm> *)
```

Comment:
This link is broken at the moment.

Reply:
Another link that may be useful is: https://en.wikipedia.org/wiki/Operator-precedence_parser#Pratt_parsing

Reply:
The link seems permanently broken. Thanks to remove it